gnodeb/context/gnbupf.go

```diff
@@ -18,8 +18,7 @@ const GTP_U_PORT int = 2152
 
 // GnbUpf holds the UPF context
 type GnbUpf struct {
-	UpfAddr     *net.UDPAddr
-	UpfIpString string
+	UpfAddr *net.UDPAddr
```

Comment:
Something similar to my previous comment, why not putting variables of the same type together? For example:
```go
	UpfAddr  *net.UDPAddr
	GnbUpUes *GnbUeDao
	Log      *logrus.Entry
```